### PR TITLE
docs: update the title of ci

### LINKED ISSUE #3167: ci: need a better title
here is the current list, it is not easy to read.

image

### PR DESCRIPTION
**Pull request by dabue**
update the title of ci

fix #3167 
### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: .github/workflows/test-e2e.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ on:
     branches: [master]
 
 jobs:
-  build:
+  e2e:
```
Comment (spacewander) [inline .github/workflows/test-e2e.yml:10]: Since our tests are not only e2e test, would it be better to use name "ci/test"?

Comment (dabue) [reply .github/workflows/test-e2e.yml:10]: agree +1

**Review by spacewander — APPROVED** [review]

**Comment by spacewander** [comment]
I found a way to let the CI run: keep the original files and then we can submit a new one to remove them after this one is merged.

File: .github/workflows/ci-test.yml [hunk 2]
```diff
@@ -0,0 +1,101 @@
+name: ci
```
Comment (spacewander) [inline .github/workflows/ci-test.yml:1]: Also need to change the badge and its Chinese version:
https://github.com/apache/apisix/blob/151d74fd9c835d9d1dc5cc7b25fb9bdbf9fc56cf/README.md#L22

**Comment by dabue** [comment]
> d then we can submit a new one to remove t

I will have a try , thanks

**Review by membphis — APPROVED** [review]

**Pull request merged**